java/fundamentals/operators-ii/prefix-and-postfix-of-increment-operators.md

```diff
@@ -16,7 +16,7 @@ Java supports 2 forms of shorter incrementation[1] of variables:
 ```java
 int i = 3;
 System.out.println(i++); 
-// 4
+// 3
```

Comment:
`// 4` was the correct value

here is the code in a REPL

![image](https://user-images.githubusercontent.com/19802557/94148495-9baec000-fe6e-11ea-8a81-7d6af3ae5462.png)

the value of `i` is incremented **after** it is printed when using postfix incrementation

Reply:
How is 4 correct?
when the outputs are 3 and 5?

Reply:
You contradicted yourself in the last sentence.

> the value of i is incremented after it is printed when using postfix incrementation

The value of i = 3
the first print prints 3 and then it increments to 4
then the next print increments to 5 and then prints

Reply:
oops, I'm blind. disregard my comment